src/ontology/efo-edit.owl

```diff
@@ -68997,7 +68997,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NCIT:C34447"^^xsd:string) obo
 AnnotationAssertion(terms:conformsTo efo:EFO_0000181 <http://purl.obolibrary.org/obo/mondo/patterns/squamous_cell_carcinoma.yaml>)
```

Comment:
Since the count here was so small, I just did these edits by hand. I admit, instead of using an actual OWL tool, I've been scripting edits with Python regexp for other repos where I contributed changes to xrefs...